src/Compilers/CSharp/Portable/Symbols/Extensions/ReceiverParameterSymbol.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed class ReceiverParameterSymbol : RewrittenParameterSymbol
```

Comment:
> ReceiverParameterSymbol

This name sounds very generic but it's only used for extensions. Should that be reflected in the name or at least the namespace?

Reply:
I am fine with the name. I do not think there is other notion of receiver parameter in the language.